base/monit.supervisord.conf

```diff
@@ -0,0 +1,5 @@
+[program:monit]
```

Comment:
I'm pretty sure the issues were around missing a true init process, but I can't find the details.

As well as the testing (which I think needs to be reasonably general), my only other concern with monit is whether this moves us further away from the intended container architecture (where containers are lightweight rather than essentially VMs).

If we're confident we should go the monit route, should we be getting rid of supervisord?